File: Estrutura_Sequencial/Python/Ex1012.py
A, B, C = map(float, input().split())
Pi = 3.14159

print(f"TRIANGULO: {(A * C) / 2:.3f}\nCIRCULO: {Pi * pow(C, 2):.3f}\nTRAPEZIO: {((A + B) * C) / 2:.3f}\nQUADRADO: {B * B:.3f}\nRETANGULO: {A * B:.3f}")

File: Estrutura_Sequencial/JavaScript/Ex1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) {return lines.shift();};

var [A, B, C] = prompt().split(' ').map(parseFloat);

triangulo = (A * C) / 2;
circulo = 3.14159 * Math.pow(C, 2);
trapezio = ((A + B) * C) / 2;
quadrado = Math.pow(B, 2);
retangulo = A * B;

console.log("TRIANGULO: " + triangulo.toFixed(3) +
            "\nCIRCULO:", circulo.toFixed(3) +
            "\nTRAPEZIO:", trapezio.toFixed(3) +
            "\nQUADRADO:", quadrado.toFixed(3) +
            "\nRETANGULO:",retangulo.toFixed(3));